spec/lib/data_update_scripts/update_articles_cached_entities_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+require Rails.root.join("lib/data_update_scripts/20200723070918_update_articles_cached_entities.rb")
+
+describe DataUpdateScripts::UpdateArticlesCachedEntities do
```

Comment:
Spec for an update script?! 👏

Reply:
It wasn't the first one in `spec/lib/data_update_scripts`, just following the good example set by @mstruve  😃